src/grasp_poses_visualizer.cpp

```diff
@@ -121,11 +121,11 @@ class GraspPosesVisualizer
     Eigen::Vector3d palm_vector = obj_point - grasp_point;
     palm_vector.normalize();
 
-    Eigen::Vector3d palm_point = grasp_point + palm_vector * grasp_data_->finger_to_palm_depth_;
+    Eigen::Vector3d palm_point = grasp_point + palm_vector * grasp_data_->grasp_max_depth_;
     visual_tools_->publishLine(grasp_point, palm_point, rviz_visual_tools::GREY);
 
     Eigen::Affine3d text_pose;
-    Eigen::Vector3d text_point = grasp_point + palm_vector * grasp_data_->finger_to_palm_depth_ * 0.5;
+    Eigen::Vector3d text_point = grasp_point + palm_vector * grasp_data_->grasp_max_depth_ * 0.5;
```

Comment:
This looks like it could be simplified a lot. For instance, grasp_point is added even though grasp_pose.translation() is subtracted two lines below.

Reply:
Agreed. This change was intended to only change the variable name. AFAIK the grasp_poses_visualizer is broken